app/markets/MarketDetails.tsx

```diff
@@ -65,9 +73,44 @@ export const MarketDetails = ({ id }: MarketDetailsProps) => {
               <OutcomeBar market={market} />
             </div>
           </div>
-          {!marketModel.isClosed && (
+          <div className="px-4">
+            <ToggleGroup
+              value={tab}
+              onChange={setTab}
+              className="w-full md:w-full justify-around"
+            >
+              <div className="w-full">
+                <ToggleGroupOption
+                  size="md"
+                  value={"bet"}
+                  className="font-semibold flex justify-center"
+                >
+                  Bet
+                </ToggleGroupOption>
+              </div>
+              <div className="w-full">
+                <ToggleGroupOption
+                  size="md"
+                  value={"history"}
+                  className="font-semibold flex justify-center"
+                >
+                  History
+                </ToggleGroupOption>
+              </div>
```

Comment:
Minor refactor, non-blocking (feel free to dismiss)
```suggestion
              {["bet", "history"].map((tab) => (
                <MarketTab tabName={tab} />
              ))}
```

The `MarketTab` component would look something like:
```tsx
const MarketTab = ({ tabName }: { tabName: string }) => (
  <div className="w-full">
    <ToggleGroupOption
      size="md"
      value={tabName}
      className="capitalize font-semibold flex justify-center"
    >
      {tabName}
    </ToggleGroupOption>
  </div>
);
```

Reply:
Don't think we need a component, but I'll take the array. 👍